pages/GOR.md

```diff
@@ -1,12 +1,186 @@
 # Game of Realms
 
-The first high-stakes contest will see participants compete for the tiered membership to co-own the Gno.land blockchain.
-A series of complex technical and non-technical tasks will challenge contributors to create innovative patterns that push the chain to new limits.
-Start building the foundation for tomorrow through key smart contracts and other contributions that change our understanding of the world.
+The worldwide competition to find the best contributors to Gno.land is currently in phase one – for advanced developers only.
 
-You can start participating in the co-creation of the Game of Realms now by adding your contributions on the dedicated [GitHub page](https://github.com/gnolang/gno/issues/357) to help us formulate the challenges. We want to release the final challenges from your contributions.
+Once the necessary tools to start phase two are ready, we’ll open up the competition to newer devs and non-technical contributors.
 
-## Register  Now
+If you want to stack ATOM rewards and play a key role in the success of Gno.land and web3, read more about Game of Realms or open a [PR](https://github.com/gnolang/gno/) today.
+
+<div>
+
+<div role="tablist" aria-labelledby="tablist-1" class="tabs">
+<div class="columns-2">
+<div>
+
+## Phase I. (ongoing)
+
+- <button id="tab-1" type="button" role="tab" aria-selected="true" aria-controls="tabpanel-1">Evaluation DAO (30%)</button>
+
+- <button id="tab-2" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-2" tabindex="-1">Tutorials (80%)</button>
+
+- <button id="tab-3" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-3" tabindex="-1">Governance Module (25%)</button>
+
+</div>
+<div>
```

Comment:
We should skip this part from the website and do it directly on contracts.

Please, keep the rest of the informative things that always make sense; we'll move the dynamic part to a realm.

Reply:
another idea, we do it with pure markdown on a repo managed by devrels cc @waymobetta.

and this page is just informative + linking to the repo; and later we replace the manual repo with smart contracts.

Reply:
@moul Do you mean we should keep something like we currently have here: https://gno.land/game-of-realms + link ?